guice/build.gradle

```diff
@@ -68,6 +68,8 @@ dependencies {
     implementation 'io.zipkin.reporter2:zipkin-sender-okhttp3:2.7.7'
 
     implementation 'io.prometheus:prometheus-metrics-exporter-servlet-javax:1.2.0'
+    implementation 'com.codahale.metrics:metrics-core:3.0.2'
```

Comment:
Can we instead use the latest version of metrics? `libraries.dw_metrics` is already included.